buildengine/build_kotlin_test.go

```diff
@@ -383,3 +387,20 @@ fun nothing(context: Context): Unit = throw
 		assertGeneratedModule("generated-sources/ftl/test/Test.kt", expected),
 	})
 }
+
+func TestKotlinExternalType(t *testing.T) {
+	moduleDir := "testdata/modules/externalkotlin"
+	buildDir := "_ftl"
+
+	ctx := log.ContextWithLogger(context.Background(), log.Configure(os.Stderr, log.Config{}))
+	module, err := LoadModule(ctx, moduleDir)
+	assert.NoError(t, err)
+
+	sch := &schema.Schema{}
+	err = Build(ctx, sch, module)
+	//Expects: Expected module name to be in the form ftl.<module>, but was com.google.type.DayOfWeek"
+	assert.Error(t, err)
```

Comment:
I wanted to assert that the error contains the expected error string, but couldn't figure out how.
`err` doesn't seem to have any real info about the error, but the logger did print out the error.

Reply:
`assert.EqualError()` is what you want

Reply:
The actual error string is not included in the error though. It is logged out rather than passed out through the error

Reply:
Ah, that wasn't clear. For that your best bet is to add a Logger that writes to a `bytes.Buffer` to the context.